src/load-config-meta.ts

```diff
@@ -24,14 +24,9 @@ export const loadConfigMeta = async () => {
   let stencilNamespace = DEFAULT_NAMESPACE;
   let stencilEntryPath = DEFAULT_STENCIL_ENTRY_PATH;
 
-  // Stencil config and Playwright config should always live next to each other, so we
-  // can assume the Stencil config lives in the same directory as the current process' execution context.
-  //
-  // From what I can tell, Playwright's CLI does not allow you to pass a path to a config, so no concerns with being in the right
-  // directory when executing tests.
-  //
-  // Use a glob pattern so we can account for both .ts and .js config files
-  const stencilConfigPath = globSync(`${process.cwd()}/stencil.config.{ts,js}`)?.[0];
+  // Find the Stencil config file in either the current directory, or the nearest ancestor directory.
+  // This allows for the Playwright config to exist in a different directory than the Stencil config.
+  const stencilConfigPath = await findUp(['stencil.config.ts', 'stencil.config.js']);
```

Comment:
While the name `stencil.config.ts` is what we use in almost all examples, it would be technically possible to to name the config file differently which would cause confusion. Should we note this limitation in the `README.md` and clearly state that Stencil config files must follow this name pattern?

Reply:
FWIW, I think either is of those 2 options is acceptable here 👍

Reply:
I'll list this as a limitation in the readme & docs for now. Only other solutions that come to mind would be:
- A CLI for the adapter that would allow you to specify a path to the Stencil config and invoke the Playwright CLI under the hood
- A `process.env` variable for the Stencil config path.

I think both of those are a bit overkill/out-of-scope for now, though.